.github/workflows/documentation.yml

```
@@ -0,0 +1,46 @@
+name: Build and publish docs to GitHub Pages
+
+on:
+  push:
+    branches:
+      # TODO: change to "master" before merging
+      - feature-docs
```

This needs to be changed before merging:

```suggestion
      - master
```